package.json

```diff
@@ -54,6 +54,7 @@
         "decimal.js": "10.1.1",
         "devbridge-autocomplete": "^1.2.20",
         "dompurify": "^1.0.10",
+        "eve": "git+https://github.com/adobe-webplatform/eve.git#eef80ed",
```

Comment:
Why added this?

Reply:
Because there were issues with installation in CI and adding this as a direct dependency fixed those. Otherwise we won't have tests running for PRs

Reply:
@habemuscode I think this is related to the Issue that you had recently that was giving an error when trying to download a git dependency and you ended up using the `/tarball/` url.